### DISCUSSION TITLE
[Wasm] Browser right-click

### DISCUSSION THREAD

**Discussion by BertrandDeSaintBezier** [unoplatform/uno#13929] [Q&A] [answered]
Hi ! 

I was wondering if (for wasm) there was a way to access the browser's default right-click functionality, such for saving an image. This could provide a workaround in certain instances for issue #13892.

![image](https://github.com/unoplatform/uno/assets/45963857/cdbdb524-78a8-46c3-ba1a-f3d721a79c90) ![image](https://github.com/unoplatform/uno/assets/45963857/079a2ced-01fe-4770-bcf0-11c901eaafa6)

**Comment by jeromelaban** [comment, marked as answer]
The behavior of the right click is disabled by default using [css styles](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/WasmCSS/Uno.UI.css). It's not the recommended way at this point though, regardless of the issue you've opened.
